stdlib/ballerina-builtin/src/main/ballerina/ballerina/jwt/validator.bal

```diff
@@ -33,56 +33,78 @@ public struct JWTValidatorConfig {
 @Return {value:"boolean: If JWT token is valied true , else false"}
 @Return {value:"Payload: If JWT token is valied return the JWT payload"}
 @Return {value:"error: If token validation fails"}
-public function validate (string jwtToken, JWTValidatorConfig config) returns (boolean, Payload, error) {
-    error err;
+public function validate (string jwtToken, JWTValidatorConfig config) returns (boolean, Payload|null) | error {
     string[] encodedJWTComponents;
-    encodedJWTComponents, err = getJWTComponents(jwtToken);
-    if (err == null) {
-        Header header;
-        Payload payload;
-        header, payload, err = parseJWT(encodedJWTComponents);
-        if (err == null) {
-            boolean isValid;
-            isValid, err = validateJWT(encodedJWTComponents, header, payload, config);
-            if (isValid) {
-                return true, payload, null;
-            }
+    match getJWTComponents(jwtToken) {
+        string[] encodedJWT => encodedJWTComponents = encodedJWT;
+        error e => return e;
+    }
+
+    Header header;
+    Payload payload;
+    match parseJWT(encodedJWTComponents) {
+        error e => return e;
+        (Header, Payload) result => {
+            var (h,p) = result;
```

Comment:
Can we use
`(header, payload) = result;`
here?

Reply:
```
var (header, payload) = result
```
 can be used i think ... I'll change it